### ISSUE TITLE
I cannot see anything new in the console

### ISSUE THREAD

**Issue by josefsabl** [jeffhatz/AJAX-Debugger#2]
I am not sure but the extension does not seem to be working. I have it enabled and there is nothing new in console while I click ajax links.

Am I missing something?

**Comment by jeffhatz** [comment]
josefsabl,

You need to have the Dev Tools open (F12, Ctrl/Cmd+Alt+I, or hamburger menu > Tools > Developer Tools). From there, click the Console tab and you will see ajax requests logged there. If the Dev Tools were open when you installed AJAX Debugger, close and reopen the tools for the extension to be loaded in.

Hope this helps!

**Comment by simplenotezy** [comment]
I don't see anything on this page: http://getactivelondon.com/activity_finder.asp?section=000100010001&activity=00013000000108930002&location=Postcode%2C+Borough%2C+Town&keyword=Date%2C+Beginner%2C+etc...&x=20&y=37

Any idea why?

**Comment by dakusan** [comment]
Are you not seeing anything at all, or are you seeing just a single AJAX call log? Did you follow his advice from "Jul 16, 2014"?

**Comment by solucionesuno** [comment]
not works. i can´t see anything in cosole. works with POST and GEST calls?

**Comment by dakusan** [comment]
Yes, it works for any AJAX call. Make sure the web page has focus and not the developer console. If the developer console has focus it doesn't catch the AJAX request. I've been meaning to fix this for ages but not got around to it. Whenever I get around to it, It'll be on my fork of the project.

There may be other fixes in my fork that may fix your problem, but it's been a few years since I've looked at this project.

**Comment by kitovyj** [comment]
I don't see anything for GET $ajax calls (POST works for some reason). I'm debugging upyourstryle.ru.
